gatsby-node.js

```diff
@@ -8,256 +8,268 @@ const { createRemoteFileNode } = require('gatsby-source-filesystem');
  * Validate plugin options
  */
 exports.pluginOptionsSchema = ({ Joi }) => {
-	return Joi.object().keys({
-		url: Joi.string().required(),
-		auth: Joi.object()
-			.keys({
-				token: Joi.string(),
-				email: Joi.string(),
-				password: Joi.string(),
-			})
-			.with('email', 'password')
-			.with('password', 'email')
-			.xor('token', 'email'),
-		type: Joi.object()
-			.keys({
-				name: Joi.string(),
-				field: Joi.string(),
-			})
-			.optional(),
-		dev: Joi.object().keys({
-			refresh: [Joi.number(), Joi.string()],
-		}),
-		graphql: Joi.object(),
-	});
+  return Joi.object().keys({
+    url: Joi.string().required(),
+    auth: Joi.object()
+      .keys({
+        token: Joi.string(),
```

Comment:
Lets keep the original tabs based formatting. We can copy paste the editorconfig/prettierconfig/eslint rules from the main Directus project to prevent this going forwards 🙂

Reply:
I got the tabs-based formatting back in order. Copied editorconfig/prettierconfig/eslint rules from the main Directus project.